src/chai/managed_ptr.hpp

```diff
@@ -664,7 +664,7 @@ namespace chai {
       ///
       template <typename T>
       CHAI_HOST_DEVICE T* getRawPointers(ManagedArray<T> arg) {
-         return (T*) arg;
+         return arg.getActivePointer();
```

Comment:
This is incorrect. `getActivePointer()` does not move data to the correct space. I have a pull request in that adds the "data" method that does the same thing as the cast. Once that is in, you can safely use that here. `return arg.data();`

Reply:
Hi @adayton1 . Should be fixed